app/ts/components/SendEthPage.tsx

```diff
@@ -46,30 +48,168 @@ const Page = ({ children }: { children: ComponentChildren }) => {
 		<Layout.Page class='grid [grid-template-areas:_"header"_"body"_"footer"] grid-rows-[minmax(min-content,max-content)_minmax(min-content,auto)_minmax(min-content,max-content)]'>{children} </Layout.Page>
 	)
 }
+
 const Header = ({ children }: { children: ComponentChildren }) => <Layout.Header class='grid [grid-template-areas:_"brand"_"profile"] md:[grid-template-areas:_"brand_profile"] md:items-center md:justify-between gap-4 py-4 px-6'>{children}</Layout.Header>
 
 const Body = ({ children }: { children: ComponentChildren }) => <Layout.Body class='grid gap-6 [grid-template-areas:_"essentials"_"main"_"aside"] md:[grid-template-areas:_"main_essentials"_"main_aside"] md:grid-cols-[3fr_2fr] xl:grid-cols-[5fr_2fr] md:grid-rows-[minmax(min-content,max-content)_1fr] px-6'>{children}</Layout.Body>
 
-const Main = () => <div class='grid [grid-template-areas:_"title"_"token"_"amount"_"address"_"tip"_"controls"] grid-rows-[repeat(auto-fit,minmax(min-content,0))] gap-y-4 lg:[grid-template-areas:_"title_title"_"token_amount"_"address_address"_"tip_tip"_"controls_controls"] lg:gap-x-6 xl:[grid-template-areas:_"title_title_title"_"token_amount_tip"_"address_address_tip"_"controls_controls_tip"]'>
-	<div class='[grid-area:title]'>
-		<div class='bg-white/10 text-xl font-bold px-6 py-2 -ml-6'>Send Funds</div>
-	</div>
-	<div class='[grid-area:token]'>
-		<TokenSelectField />
-	</div>
-	<div class='[grid-area:amount]'>
-		<AmountField value='0' onChange={() => { }} label='Amount' name='amount' />
-	</div>
-	<div class='[grid-area:address]'>
-		<AddressField value='' onChange={() => { }} label='Address' name='to' />
-	</div>
-	<div class='[grid-area:tip] border border-dashed border-white/30'>
-		<div class='p-4'>Guide</div>
-	</div>
-	<div class='[grid-area:controls]'>
-		<button class='px-4 py-2 hover:bg-white/10 border w-full'>Send</button>
-	</div>
-</div>
+const Main = () => {
+	return (
+		<SendForm>
+			<div class='grid [grid-template-areas:_"title"_"token"_"amount"_"address"_"tip"_"controls"] grid-rows-[repeat(auto-fit,minmax(min-content,0))] gap-y-4 lg:[grid-template-areas:_"title_title"_"token_amount"_"address_address"_"tip_tip"_"controls_controls"] lg:gap-x-6 xl:[grid-template-areas:_"title_title_title"_"token_amount_tip"_"address_address_tip"_"controls_controls_tip"] lg:grid-cols-2 xl:grid-cols-3'>
```

Comment:
hmm this feels quite ugly, could this be a single or couple css classes instead?

Reply:
yeah, it's indeed a quick and dirty approach to building the UI. these stuff later gets optimized outside of the utility styles tailwind generates. creating a separate issue for cleaning this up 👍 #56